Impwreck Content Pack 1

Sleiphros opened the pull request:
<!-- To be specific: 12 new wrecks (5 small/5 medium/2 large) and 6 new ruins/derelicts-->
<!-- I have been staring at this screen for half an hour. This is so scary. -->

**Changelog**
:cl: Sleiphros
- add: New wrecks of all different sizes can be pulled in by the salvage magnet!
- add: New derelicts, ruins and satellites have been spotted floating near the station recently...

Darkmajia commented:
i had a look over each map and i have a few thoughts:

general note:
- for spaced areas, you can use "Atmos Fix Vacuum Marker" to prevent air from filling those tiles with fixgridatmos: this can save a little atmos performance at roundstart. probably worth passing through the maps to make sure these are put down
- might be worth double-checking that external airlocks have directional fans under them, though i realise it may be intentional that some of these maps don't have them placed

cozy-radio-planetoid:
- tiny fans have been phased out in favour of directional fans, might be better to find a different atmos solution for this ruin or leave it spaced, though i realise what the intention is and how it'd be nice to take the hardsuits off

science-satellite:
- has no atmos even in the enclosed rooms, though i assume this is intentional?

foggy-meat-town:
- the air pressure is crazy and damages people in spationaut suits - maybe use the decapoid atmosphere atmos markers instead of the regular ones

overall though, i think these are great. love to see them

Sleiphros commented:
things are screwy on my end. lemme try redoing this pull request. in the meantime to address those thoughts:
- added the vacuum fixer markers to maps that needed it
- added directional fans to a few docking ports that got missed (such as in snowy-sauna)
- fixed foggy-meat-town's vapor atmos. there were points where it was layered multiple times over, up to four times in some spots? never trusting Replace again on non-wall objects
- switched up cozy-radio-planetoid's outer ring to use directional fans
- science-satellite initially didn't have an atmosphere, thought about it and gave it some residual air and a lot of vacuum tiles
